CONTRIBUTING.md

```diff
@@ -1,21 +1,22 @@
+# Contributing
+
 # Issues
-Before opening an issue, please make sure your problem/request doesn't exist. When you open issue, please provide the following information if possible:
+Before opening an issue, please make sure your problem or request doesn't exist. When opening an issue, please provide the following information if possible:
 
 * Example code or repo (please keep it simple and minimal)
-* Steps to reproduce
-* `react-bootstrap-table2` version
+* Steps to reproduce.
+* `react-bootstrap-table2` version.
 
-Anyway, you're welcome to open an issue to ask questions or discuss about a feature request.
+Anyway, you're always welcome to open an issue to ask questions or talk about new feature request.
 
 # Pull Requests
 Check [here](./docs/development.md) for getting started with development.
 
-* When you want to implement a new feature, please let us know (via issues).
-* Please run tests before opening a PR and also remember to test the actual business logic.
-* If your PR is trying to fix a bug, please describe the bug number with hashtag when creating the PR.
+* We recommend filing an [issue](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues) before you implement any new features.
+* Please ensure all the test suit passes before submitting a PR. And always test the actual business logic.
```

Comment:
Please ensure that all the test suites have passed before submitting a PR. Besides, always test the actual business logic.

Reply:
Yeah, this looks better :)